week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -1,40 +1,42 @@
-/*
-Many years into the future, a team of Space Voyagers find their ship is low on Oxygen and need to dock
-somewhere safe while they call home for help.
-  
-Their computer detects a list of nearby planets that have Oxygen in their atmosphere.
-
-To be safe, they need to land on the first unamed planet that has Oxygen levels between 19.5% and 23.5%.
-
-Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
-*/
-
-function safeLevels() {
-
-}
-
-/* ======= TESTS - DO NOT MODIFY ===== */
-
-const oxygenLevels1 = ["24.2%", "11.3%", "19.9%", "23.1%", "29.3%", "20.2%"]
-const oxygenLevels2 = ["30.8%", "23.5%", "18.8%", "19.5%", "20.2%", "31.6%"]
-
-function test(test_name, expr) {
-    let status;
-    if (expr) {
-      status = "PASSED";
-    } else {
-      status = "FAILED";
-    }
-  
-    console.log(`${test_name}: ${status}`);
-}
-
-test(
-    "safeLevels function works - case 2",
-    safeLevels(oxygenLevels1) === "19.9%"
-);
-
-test(
-    "safeLevels function works - case 2",
-    safeLevels(oxygenLevels2) === "20.2%"
-);
\ No newline at end of file
+// /*
+// Many years into the future, a team of Space Voyagers find their ship is low on Oxygen and need to dock
+// somewhere safe while they call home for help.
+
+// Their computer detects a list of nearby planets that have Oxygen in their atmosphere.
+
+// To be safe, they need to land on the first unamed planet that has Oxygen levels between 19.5% and 23.5%.
+
+// Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
+// */
+
+// function safeLevels(arr) {
+//   return arr.find(element => element > "19.5%" && element < "23.5%")
```

Comment:
Looks like this has been commented out for some reason?

Your solution is really close, but I don't think this is quite right.

The problem here is that you're using the greater than and less than operators with strings not numbers.

Try changing one of the values in the test to "100.5%". Does it do what you expected? How could you fix it?